packages/inter-protocol/src/proposals/demoIssuers.js

```diff
@@ -462,33 +462,24 @@ export const poolRates = (issuerName, record, kits, central) => {
  *   consume: { mints }
  * }} powers
  */
+// XXX Is this dead? I can't find references to it. It seems to do more than
```

Comment:
in master it's only used in `SIM_CHAIN_MANIFEST`. @michaelfig is it still needed for that? Does green CI imply "no"? 

Either way this isn't XXX tech debt that should go into master. I'd make this comment DONOTMERGE and make sure it's resolved before approval.